gateway/proxy_eth.go

```diff
@@ -498,7 +498,7 @@ func (gw *Node) EthNewPendingTransactionFilter(ctx context.Context) (ethtypes.Et
 		return ethtypes.EthFilterID{}, err
 	}
 
-	return addUserFilterLimited(ctx, func() (ethtypes.EthFilterID, error) {
+	return gw.addUserFilterLimited(ctx, "EthNewPendingTransactionFilter", func() (ethtypes.EthFilterID, error) {
```

Comment:
I'm not sure that's going to improve maintainability because it'll separate the method names from the methods themselves. This way they are at least close.